pkg/upload/android-aab.go

```diff
@@ -48,26 +49,29 @@ func ProcessAndroidAab(apiKey string, applicationId string, buildUuid string, pa
 			}
 
 			log.Success(filepath.Base(path) + " expanded")
-
-			aabManifestPath = filepath.Join(tempDir, "base", "manifest", "AndroidManifest.xml")
-
 		} else {
 			return fmt.Errorf(path + " is not an AAB file")
 		}
 	}
 
+	if aabManifestPath == "" {
+		aabManifestPathExpected := filepath.Join(tempDir, "base", "manifest", "AndroidManifest.xml")
+		if utils.FileExists(aabManifestPath) {
+			aabManifestPath = aabManifestPathExpected
+			log.Info("Found app manifest at: " + aabManifestPath)
```

Comment:
I'd remove this log message as it's in the temp dir, so not a path that means anything to someone with an AAB